src/main/main.ts

```diff
@@ -74,6 +80,11 @@ const setWindowDimension = (isExpanded: boolean) => {
 const state: IState = fetchSettings();
 let configPath = '';
 
+// create presets dir if it doesn't exist
+if (!fs.existsSync(PRESETS_DIR)) {
```

Comment:
The node docs aren't clear but I think fs functions will throw exceptions. https://stackoverflow.com/questions/60497053/exception-in-node-when-using-mkdirsync also suggests try catch. Can we catch exceptions here?

Reply:
Just to clarify, are you referring to the `fs.existsSync` function or the `fs.mkdirSync` function? (or both...?)

Reply:
Sure, we can't preset if presets folder fails to create though. Added try catch but still throw e if it happens